app/assets/javascripts/base.js.erb

```diff
@@ -57,7 +57,7 @@ var instantClick
     return target
   }
 
-  function isBlacklisted(elem) {
+  function isForbiddenFromPreloading(elem) {
```

Comment:
@rhymes this is purely a personal opinion but this function name seems a bit clunky, what about something like `isForbiddenFromPreload`  or `isNotPreloadable`

Reply:
Make sense, i'll change it! thank you @Ridhwana!